### PR TITLE
makes admin player counter ignore drones

### PR DESCRIPTION
**Pull request by silicons**
Drones are not a good thing to include in an approximation of how much of the server is dead/alive/antag.

### CODE REVIEW

File: code/modules/admin/player_panel.dm [hunk 1]
```diff
@@ -373,7 +377,7 @@
 					other_players++
 		dat += "<BR><b><font color='blue' size='3'>Players:|[connected_players - lobby_players] ingame|[connected_players] connected|[lobby_players] lobby|</font></b>"
 		dat += "<BR><b><font color='green'>Living Players:|[living_players_connected] active|[living_players - living_players_connected] disconnected|[living_players_antagonist] antagonists|</font></b>"
-		dat += "<BR><b><font color='#bf42f4'>SKIPPED \[On centcom Z-level\]: [living_skipped] living players|</font></b>"
+		dat += "<BR><b><font color='#bf42f4'>SKIPPED \[On centcom Z-level\]: [living_skipped] living players|[drones] living drones|</font></b>"
```
Comment (PKPenguin321) [inline code/modules/admin/player_panel.dm]: Feels like this is the wrong way to do this, maybe just don't count them at all instead

Comment (ExcessiveUseOfCobblestone) [inline code/modules/admin/player_panel.dm]: you need to know how many you need to blow up though.

**Comment by YotaXP** [comment]
I would consider a server with 2 living humans, and 40 drones to be alive.

File: code/modules/admin/player_panel.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/admin/player_panel.dm]: Weird to have a hole in calculations though

**Pull request merged**